petab/visualize/helper_functions.py

```diff
@@ -1,18 +1,24 @@
 """
 This file should contain the functions, which PEtab internally needs for
 plotting, but which are not meant to be used by non-developers and should
-hence not be directly visible/usable when using import `petab.visualize`
-
+hence not be directly visible/usable when using `import petab.visualize`.
 """
 
+import functools
+import warnings
+
+import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
-import matplotlib.pyplot as plt
 import petab
-import functools
-import warnings
-from .plotting_config import plot_lowlevel
 import seaborn as sns
+
+from .plotting_config import plot_lowlevel
+from ..C import (SIMULATION, DATASET_ID, SIMULATION_CONDITION_ID,
```

Comment:
well.

Reply:
Beautiful? Yeah, haven't decided how to best handle it... but as still not all are replaced, it probably good to change to star-import then...